doofetch-formula/doofetch/init.sls

```diff
@@ -0,0 +1,40 @@
+{#-
+Salt state file for managing doofetch
+Copyright (C) 2024 SUSE LLC <georg.pfuetzenreuter@suse.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-#}
+
+{%- set mypillar = salt['pillar.get']('doofetch', {}) %}
+
+doofetch_package:
+  pkg.installed:
+    - name: doofetch
+
+{%- if mypillar and mypillar is mapping %}
+doofetch_sysconfig:
+  suse_sysconfig.sysconfig:
+    - name: /etc/sysconfig/doofetch
+    - quote: false
```

Comment:
What's the reason for `quote: false`?

Also, a small pillar.example would be nice.

Reply:
Thanks for adding pillar.example!

> Mostly that it's not quoted in the default sysconfig file shipped with the package.

That sounds like a bug in the package ;-) - sysconfig files are typically sourced in shell scripts, and quoting can avoid quite some problems (most boring one in this case: a directory name containing a space). Please add quotes in the packaged file and in the formula.

Reply:
Well, someone could use `targetdir: /srv/my images` which would need quotes - unless you want to execute `images` when sourcing the sysconfig file ;-)

That said: The usual `"` would be fine, but I'm also fine with explicitely specifying `'` - so feel free to merge.

Reply:
I get it, I just hope my users to not use directory names with spaces. ;-) But of course I agree it's good practice to support it anyways.

I prefer single quotes to avoid expansion (albeit that usually not happening when sourcing).